### ISSUE TITLE
Installing WooCommerce breaks out of JPO

### ISSUE THREAD

**Issue by ryelle** [Automattic/jetpack-onboarding#49]
It's a little strange to go from JPO -> WooCommerce wizard -> back to JPO.

We could keep track if the user needs to set up Woo, and then direct them into the wizard at the review step?

Let's figure out if this is necessary & how we should proceed.

**Comment by ryelle** [comment]
The interruption to install WooCommerce also makes coming back to the question about a previous store seem awkwardly placed. If we delay the WC install to after the review step, we could do this:

> Should install WooCommerce? Yes -> Do you have a past store? Yes, ___ -> Review -> Set up WooCommerce

or

> Should install WooCommerce? Yes -> Do you have a past store? Yes, ___ -> Set up WooCommerce -> Review

**Comment by tyxla** [comment]
I totally agree with you on this. It feels really weird to have that step at the end. Both suggestions look better to me, but I'd probably go with:

> Should install WooCommerce? Yes -> Do you have a past store? Yes, ___ -> Set up WooCommerce -> Review

as the Review step should be at the end, otherwise it might still look confusing to the users.

**Comment by gravityrail** [comment]
I tried addressing the redirect bug in https://github.com/Automattic/jetpack-onboarding/commit/7a65ac0e39eacfe2081a8d9607df11150f5cab5d

I didn't address the "past store" stuff.
